src/openforms/tests/test_celery_health_check.py

```diff
@@ -0,0 +1,65 @@
+import subprocess
+import time
+from pathlib import Path
+
+from django.test import TestCase
+
+from openforms.celery import app
+
+READINESS_FILE = (
+    Path(__file__).parent.parent.parent.parent / "tmp" / "celery_worker_ready"
+)
+WORKER = Path(__file__).parent.parent.parent.parent / "bin" / "celery_worker.sh"
+
+
+class CeleryTest(TestCase):
+    def setUp(self):
+        def shutdown_celery():
+            app.control.shutdown()
+            if READINESS_FILE.is_file():
+                READINESS_FILE.unlink(missing_ok=True)
+
+        self.addCleanup(shutdown_celery)
+
+    def test_celery_worker_health_check(self):
+        """Assert that READINESS_FILE exists after worker has started but not before and not after
+        the shutdown
+        """
+        assert (
+            not READINESS_FILE.is_file()
+        ), "Celery worker not started but READINESS_FILE found"
+
+        # start Celery worker
+        subprocess.Popen([WORKER], shell=True)
```

Comment:
I had the same issue running the test from within my editor because it didn't pick up the path. Works fine if run from the command line. However, it doesn't hurt adding the `cwd` for convenience, I suppose.